recipes/seqan_tcoffee/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+mkdir -p build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
```

Comment:
A general question. You also could use the `-DCMAKE_INSTALL_PREFIX` I assume?

Reply:
I think so, 
I can update line 6 with -DCMAKE_INSTALL_PREFIX=$PREFIX/bin and test it.

Reply:
Would be great if you can test this! Thanks!

Reply:
Dear @bgruening, do you have other tests or can we merge this PR?